api/v1beta1/flowcollector_types.go

```diff
@@ -672,19 +690,24 @@ type ExporterType string
 
 const (
 	KafkaExporter ExporterType = "KAFKA"
+	IpfixExporter ExporterType = "IPFIX"
 )
 
 // FlowCollectorExporter defines an additional exporter to send enriched flows to
 type FlowCollectorExporter struct {
 	// type selects the type of exporters. Only "KAFKA" is available at the moment.
 	// +unionDiscriminator
-	// +kubebuilder:validation:Enum:="KAFKA"
+	// +kubebuilder:validation:Enum:="KAFKA";"IPFIX"
 	// +kubebuilder:validation:Required
 	Type ExporterType `json:"type"`
 
 	// kafka configuration, such as address or topic, to send enriched flows to.
 	// +optional
 	Kafka FlowCollectorKafka `json:"kafka,omitempty"`
+
+	// ipfix configuration, such as ip address and port to send ipfix flows to.
+	// +optional
+	Ipfix FlowCollectorIPFIXReceiver `json:"ipfix,omitempty"`
```

Comment:
nit; using uppercase for accronyms

```suggestion
	IPFIX FlowCollectorIPFIXReceiver `json:"ipfix,omitempty"`
```